conformance/operator_test.go

```diff
@@ -290,14 +292,15 @@ var _ = Describe("operator", func() {
 				stdout, stderr, err := pod.ExecCommand(clients, podObj, "ip", "addr", "show", "dev", "net1")
 				Expect(err).ToNot(HaveOccurred())
 				Expect(stderr).To(Equal(""))
-				podMac := getMacFrom(stdout)
+				vfID := findVF(intf.Name, numVfs, debugPod)
+				Expect(vfID).ToNot(Equal(-1))
 				stdout, stderr, err = pod.ExecCommand(clients, debugPod, "ip", "link", "show")
 				Expect(err).ToNot(HaveOccurred())
 				Expect(stderr).To(Equal(""))
 
 				found := false
 				for _, line := range strings.Split(stdout, "\n") {
-					if strings.Contains(line, podMac) && containsFunc(line) {
+					if strings.Contains(line, fmt.Sprintf("vf %d MAC", vfID)) && containsFunc(line) {
```

Comment:
I think that now looking for MAC too may be a bit misleading. Can you just match strings starting with vf vfID ?